<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RvrSupplierEditor</class>
 <widget class="QDialog" name="RvrSupplierEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Supplier Details</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Supplier Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtSupplierName"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txtEmail"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>E-Mail Address:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Contact Name:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Address:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Tel:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="txtAddress"/>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="txtTel"/>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="txtFax"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Fax:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Town:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="txtTown"/>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="txtContactName"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Contact Email:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" rowspan="3">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Country:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" rowspan="3">
      <widget class="QLineEdit" name="txtCountry"/>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="txtContactEmail"/>
     </item>
     <item row="2" column="2" rowspan="3">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Website:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3" rowspan="3">
      <widget class="QLineEdit" name="txtWebsite"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>101</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="cmdSave">
       <property name="text">
        <string>Save</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cmdCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cmdCancel</sender>
   <signal>clicked()</signal>
   <receiver>RvrSupplierEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
